pkg/lambdaworks/lambdaworks.go

```diff
@@ -86,6 +92,15 @@ func (felt Felt) ToU64() (uint64, error) {
 	}
 }
 
+// turns a felt to usize
+func (felt Felt) ToUint() (uint, error) {
+	if felt.limbs[0] == 0 && felt.limbs[1] == 0 && felt.limbs[2] == 0 {
+		return uint(felt.limbs[3]), nil
+	} else {
+		return 0, ConversionError(felt, "uint")
+	}
+}
```

Comment:
This should call Felt.ToU64() instead of cpoying its logic

Reply:
Resolved: https://github.com/lambdaclass/cairo-vm_in_go/pull/241/commits/22c3df1ba52ba746f9f35199e4df688826b3fefe